pkg/server/workspaces/remove.go

```diff
@@ -70,3 +70,32 @@ func (s *WorkspaceService) RemoveWorkspace(workspaceId string) error {
 	log.Infof("Workspace %s destroyed", workspace.Id)
 	return nil
 }
+
```

Comment:
@Tpuljak is this the correct implementation ?

Reply:
@mobley-trent did you test this implementation?

Reply:
I tried running the tests locally but they are all failing. Do I need to run the dev environment in a Docker container ?

Reply:
I recommend opening Daytona in a dev container using VS Code.

Reply:
@Tpuljak I'm currently trying to run `go run cmd/daytona/main.go serve` after pulling the latest upstream changes but I'm getting this error, which I'm currently trying to debug:

```
daytona ➜ /workspaces/daytona (force_remove_workspace) $ go run cmd/daytona/main.go serve
DEBU[0000] Opening db at /home/daytona/.config/daytona/db 
INFO[0000] Starting Daytona server                      
INFO[0000] Starting local container registry            
Image already pulled
FATA[0002] Error response from daemon: driver failed programming external connectivity on endpoint daytona-registry (11856c18165647b8371091f1f8aa363e4310eda44c2757f97b325924c12b1a3d):  (iptables failed: iptables --wait -t nat -A DOCKER -p tcp -d 0/0 --dport 32777 -j DNAT --to-destination 172.18.0.2:0 ! -i docker0: iptables v1.8.7 (legacy): Port `0' not valid

Try `iptables -h' or 'iptables --help' for more information.
 (exit status 2)) 
exit status 1
```

Is this an issue that came up for everyone. I'm currently running the dev environment in a Docker container.

Reply:
@mobley-trent you shouldn't start daytona in a Docker container.

Simply clone the repo locally, open it in VS Code and then run the VS Code command: `Dev Containers: Open Folder in Container...`

You can find more info here https://code.visualstudio.com/docs/devcontainers/containers#_quick-start-open-an-existing-folder-in-a-container

This will set you up with everything you need to start working with Daytona.

Reply:
@Tpuljak Yes I used the VS Code command, but on trying to run `go run cmd/daytona/main.go serve`, the operation fails. This is how I used to test the code previously. Is this not the intended operation for running daytona locally ?

Reply:
Ah, I see the issue, my bad.
Please take a look at our latest release https://github.com/daytonaio/daytona/releases/tag/v0.17.0
Specifically, the Breaking changes. You will need to update the server configuration.

If the errors happens right away, I recommend you execute `code ~/.config/daytona/server/config.json` and then add `"registryPort": "3988"` to the json structure.

Reply:
@Tpuljak I ended up setting up the dev environment on Codespaces as a temporary fix. It seems to be working okay from there.

Reply:
@mobley-trent thanks for working with us on this and for the patience. We'll investigate further while you work in Codespaces.